appveyor.yml

```diff
@@ -13,29 +13,30 @@ environment:
     - python: 37-x64
     - python: 38
     - python: 38-x64
-    - { python: 27, SUBUNIT: 1 }
-    - { python: 27-x64, SUBUNIT: 1 }
-    - { python: 35, SUBUNIT: 1 }
-    - { python: 35-x64, SUBUNIT: 1 }
-    - { python: 36, SUBUNIT: 1 }
-    - { python: 36-x64, SUBUNIT: 1 }
-    - { python: 37, SUBUNIT: 1 }
-    - { python: 37-x64, SUBUNIT: 1 }
-    - { python: 38, SUBUNIT: 1 }
-    - { python: 38-x64, SUBUNIT: 1 }
+    - python: 39
+    - python: 39-x64
 
 install:
+  - "SET PYTHONVERSION=%PYTHON%"
   - "SET PATH=C:\\Python%PYTHON%;c:\\Python%PYTHON%\\scripts;%PATH%"
+  - ps: |
+      $env:PYTHON = "C:\\Python${env:PYTHON}"
+      if (-not (Test-Path $env:PYTHON)) {
+        curl -o install_python.ps1 https://raw.githubusercontent.com/matthew-brett/multibuild/11a389d78892cf90addac8f69433d5e22bfa422a/install_python.ps1
+        .\install_python.ps1
+      }
+  - ps: if (-not (Test-Path $env:PYTHON)) { throw "No $env:PYTHON" }
   - echo "C:\Program Files\Microsoft SDKs\Windows\v7.1\Bin\SetEnv.cmd" /x64 > "C:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\bin\amd64\vcvars64.bat"
-  - python -m pip install -U setuptools pip
-  # NB: pip install -e .[test] fails for obscure namespace package reasons
-  - pip install -U .[test]
-  - if %SUBUNIT%==1 pip install -U .[subunit]
+  - python -m pip install -U pip
+  - pip install -U setuptools wheel
+  - pip install -U -e .[test]
 
 build: false
+matrix:
+  fast_finish: true
 
 test_script:
-  - python setup.py -q test -q
+  - zope-testrunner --test-path=src
```

Comment:
Hmm.  Can we rely on zope.testrunner to run its own tests without relying on whatever hacks `setup.py test` does?

Reply:
The consensus seem to be "probably"?